### DISCUSSION TITLE
Does APISIX Plugin support using var in configurations?

### DISCUSSION THREAD

**Discussion by chenjizheng** [apache/apisix#6205] [Ideas]
希望插件配置值支持变量获取，如uri：${ctx.var.uri}

**Comment by juzhiyuan** [comment]
Hi @chenjizheng, just to make sure, you'd like to use vars in your custom plugin? AFAIK, the traffic-split[1] plugin supports using vars, you could have a check how it implement. See https://github.com/apache/apisix/blob/master/apisix/plugins/traffic-split.lua

Also cc @tzssangglass 

[1] https://apisix.apache.org/docs/apisix/plugins/traffic-split

**Comment by tzssangglass** [comment]
Need to describe more detailed.
